### PR TITLE
Edit README.md

### PR DESCRIPTION
**Pull request by IAmTheRedSpy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,15 +4,15 @@
 [![goto counter](https://img.shields.io/github/search/tbodt/ish/goto.svg)](https://github.com/tbodt/ish/search?q=goto)
 [![fuck counter](https://img.shields.io/github/search/tbodt/ish/fuck.svg)](https://github.com/tbodt/ish/search?q=fuck)
 
-A project to get a Linux shell running on iOS, using usermode x86 emulation and syscall translation.
+iSH is a Linux shell on iOS with usermode x86 emulation and syscall translation.
```
Comment (tbodt) [inline README.md]: I prefer the earlier wording:
```suggestion
iSH is a Linux shell running iOS, using usermode x86 emulation and syscall translation.
```

File: README.md [hunk 2]
```diff
@@ -28,9 +28,9 @@ You can replace `ish` with `tools/ptraceomatic` to run the program in a real pro
 
 To compile the iOS app, just open the Xcode project and click run. There are scripts that should download and set up the alpine filesystem and create build directories for cross compilation and so on automatically.
 
-## Further setup guide
+## Development setup
```
Comment (tbodt) [inline README.md]: These steps are only necessary for building and running the iOS app.
```suggestion
## iOS development setup
```

File: README.md (hunk 1 above)
Comment (tbodt) [reply README.md]: Ah yeah, updated the suggestion.

Comment (IAmTheRedSpy) [reply README.md]: "Running on" seems implied to me but I'm okay with the extra clarity of specifying.

"With" captures how x86 emulation and syscall translation are features of iSH but "using" implies they are from another library.

Comment (tbodt) [reply README.md]: How about:
```suggestion
iSH is a Linux shell running on iOS done with usermode x86 emulation and syscall translation.```

Comment (IAmTheRedSpy) [reply README.md]: I'd prefer
```suggestion
iSH is a Linux shell on iOS featuring usermode x86 emulation and syscall translation.
```
or keeping it the same as before.

Comment (tbodt) [reply README.md]: "x86 emulation and syscall translation" isn't really a list of features, it's more of a description of how it's accomplished.

Comment (IAmTheRedSpy) [reply README.md]: Alright, "done" seems appropriate
```suggestion
iSH is a Linux shell running on iOS, done with usermode x86 emulation and syscall translation.
```

**Comment by tbodt** [comment]
Closing due to inactivity.

**Pull request closed without merging**